setup.py

```diff
@@ -81,6 +81,7 @@ def run(self):
     "dpath>=2.0.1,<3",
     "shtab>=1.3.0,<2",
     "rich>=3.0.5",
+    "methodtools>=0.4.2",
```

Comment:
Doesn't seem like there is a conda package for it :slightly_frowning_face:  Need to use something else. Surprized you don't have something like this in funcy :wink:

Reply:
Why not just functool's lru_cache, btw? Was there a particular reason not to use it @Suor ?